warp-tori/src/main/java/com/schibsted/nmp/warp/brands/tori/ToriColors.kt

```diff
@@ -199,6 +200,12 @@ internal object ToriComponentColors : WarpComponentColors {
     override val navBar: WarpNavBarColors = ToriNavBarColors
     override val tooltip: WarpTooltipColors = ToriTooltipColors
     override val switch: WarpSwitchColors = ToriSwitchColors
+    override val spinner: WarpSpinnerColors = ToriSpinnerColors
+}
+
+private object ToriSpinnerColors : WarpSpinnerColors {
```

Comment:
No need to create an object for these as the colors used are not custom component tokens, but already existing semantic color tokens. The spinner colors are defined in https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?gid=1588196525#gid=1588196525